dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ws/metadata/ejb/EJBMDOrchestrator.java

```diff
@@ -1296,6 +1296,9 @@ private MethodDataWrapper initializeBeanMethodMD(boolean isEntity,
                                                                               methodInterface); //d599046
         }
 
+        //Set a flag if any method is annotated with the concurrency Asychronous annotation.
+        bmd.ivHasConcurrentAsynchMethod |= MethodAttribUtils.getConcurrentAsynchronousMethods(ejbMethods);
```

Comment:
Per the above comment, there is no need to save this in bmd; just call the method (perhaps renamed) and have the method log the error and throw the exception.